### PR TITLE
Update to Distributed Minio Quickstart Guide

### PR DESCRIPTION
**Pull request by jamesneimanconsulting**
This is a complete copy edit of the guide.

- [X ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by deekoder** [comment]
Needs erasure coded guide to be merged first.

File: docs/distributed/README.md [hunk 1]
```diff
@@ -1,74 +1,85 @@
 # Distributed Minio Quickstart Guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io) [![Go Report Card](https://goreportcard.com/badge/minio/minio)](https://goreportcard.com/report/minio/minio) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/minio.svg?maxAge=604800)](https://hub.docker.com/r/minio/minio/) [![codecov](https://codecov.io/gh/minio/minio/branch/master/graph/badge.svg)](https://codecov.io/gh/minio/minio)
 
-Minio in distributed mode lets you pool multiple drives (even on different machines) into a single object storage server. As drives are distributed across several nodes, distributed Minio can withstand multiple node failures and yet ensure full data protection.
+When running Minio in distributed mode (_distributed Minio_), multiple drives from different machines can be pooled into a single object storage server. As drives are distributed across several nodes, distributed Minio ensures full data protection when multiple node failures occur.
```
Comment (eco-minio) [inline docs/distributed/README.md]: "into a single storage server" might be confusing for some users, maybe single namespace is better here? @deekoder

File: docs/distributed/README.md [hunk 2]
```diff
@@ -1,74 +1,85 @@
...
 
-## Why distributed Minio?
+## Why Distributed Minio?
 
-Minio in distributed mode can help you setup a highly-available storage system with a single object storage deployment. With distributed Minio, you can optimally use storage devices, irrespective of their location in a network.
+Running Minio in distributed mode allows for the deployment of a high-availability, object storage system, enabling optimal use of storage devices regardless of their location on a network.
```
Comment (eco-minio) [inline docs/distributed/README.md]: "regardless of their location on a network" needs to be said in such a way that it implies that each cluster should reside in the same network (but multiple clusters from multiple networks can be federated).

File: docs/distributed/README.md [hunk 3]
```diff
@@ -1,74 +1,85 @@
...
+Distributed Minio provides protection against multiple node/drive failures and [bit rot](https://docs.minio.io/docs/minio-erasure-code-quickstart-guide.html#whatisrot) using [erasure code](https://docs.minio.io/docs/minio-erasure-code-quickstart-guide). Since distributed Minio and erasure code both require a minimum of four hard drives, erasure code is automatically used when distributed Minio is launched.
 
-### High availability
+### High Availability
 
-A stand-alone Minio server would go down if the server hosting the disks goes offline. In contrast, a distributed Minio setup with _n_ disks will have your data safe as long as _n/2_ or more disks are online. You'll need a minimum of _(n/2 + 1)_ [Quorum](https://github.com/minio/dsync#lock-process) disks to create new objects though.
+A standalone server running Minio Server will go down if the hard drives go offline. In contrast, a distributed Minio configuration with N disks will keep data safe as long as N/2 or more hard drives are online. This requires a minimum of (N/2 + 1) [Quorum](https://github.com/minio/dsync#lock-process) hard drives to create new objects.
```
Comment (eco-minio) [inline docs/distributed/README.md]: Did we finalize on whether to capitalize server?

File: docs/distributed/README.md [hunk 4]
```diff
@@ -1,74 +1,85 @@
...
 
-For example, an 8-node distributed Minio setup with 1 disk per node would continue serving files, even if up to 4 disks are offline. But, you'll need at least 5 disks online to create new objects.
+For example, an 8-node distributed Minio configuration with one hard drive per node will continue serving files even if as many as 4 hard drives are offline. This configuration requires at least 5 hard drives online to create new objects.
 
 ### Limits
 
-As with Minio in stand-alone mode, distributed Minio has a per tenant limit of minimum 2 and maximum 32 servers. There are no limits on number of disks shared across these servers. If you need a multiple tenant setup, you can easily spin up multiple Minio instances managed by orchestration tools like Kubernetes.
+As with Minio running in standalone mode, distributed Minio supports 2-32 servers per tenant. There are no limits on the number of hard drives shared across these servers. For a multi-tenant configuration, orchestrate multiple Minio instances using a tool like Kubernetes.
```
Comment (eco-minio) [inline docs/distributed/README.md]: This sentence is confusing as it implies that standalone uses more than one server.  Should also note that Minio itself does not impose limits on the number of hard drives.

File: docs/distributed/README.md [hunk 5]
```diff
@@ -1,74 +1,85 @@
...
 
-Note that with distributed Minio you can play around with the number of nodes and drives as long as the limits are adhered to. For example, you can have 2 nodes with 4 drives each, 4 nodes with 4 drives each, 8 nodes with 2 drives each, 32 servers with 24 drives each and so on.
+**Note:** With distributed Minio, the number of nodes and hard drives can be adjusted within the limits. For example, a configuration can have 2 nodes with 4 hard drives each, 4 nodes with 4 hard drives each, 8 nodes with 2 hard drives each, 32 servers with 24 hard drives each, etc.
```
Comment (eco-minio) [inline docs/distributed/README.md]: Need to replace adjusted, it implies a cluster can have nodes added and removed.

File: docs/distributed/README.md [hunk 6]
```diff
@@ -1,74 +1,85 @@
...
+To start a distributed Minio instance:
+1. Pass the hard drive locations as parameters to the `minio server` command.
+2. Re-run the same command on all of the participating nodes.
+
+**Note:**
+* All nodes running distributed Minio must have the same access key and secret key for the nodes to connect. This requires that the access key and secret key be exported as the following environment variables on all nodes before executing the `minio server` command: `MINIO_ACCESS_KEY` and `MINIO_SECRET_KEY`.
+* All nodes running distributed Minio must be in a homogeneous environment (meaning they have the same operating system, number of drives, and interconnects).
+* If the domain must be set, the `MINIO_DOMAIN` environment variable must be defined and exported.
+* Minio distributed mode requires directories that are exclusive to Minio. For example, if a volume is mounted under `/export`, pass `/export/data` as an argument to the `minio server` command.
+**Note:** The drives can be shared with other applications if required.
```
Comment (eco-minio) [inline docs/distributed/README.md]: "The drives can be shared with other applications" - This isn't clear, we should either point to a way to do this or remove it as it is too ambiguous.  @deekoder thoughts?

File: docs/distributed/README.md [hunk 7]
```diff
@@ -1,74 +1,85 @@
...
 ```
 
+**Note:** Arguments such as `{1...n}` have three dots. Using only two dots (e.g. `{1..4}`) will be interpreted by the shell and won't be passed to Minio Server. This affects the erasure coding order, possibly impacting performance and high availability. Always use `{1...n}` (three dots) to use Minio Server erasure code on data optimally.
 
-__NOTE:__ `{1...n}` shown have 3 dots! Using only 2 dots `{1..4}` will be interpreted by your shell and won't be passed to minio server, affecting the erasure coding order, which may impact performance and high availability. __Always use `{1...n}` (3 dots!) to allow minio server to optimally erasure-code data__
-
-## 3. Test your setup
-To test this setup, access the Minio server via browser or [`mc`](https://docs.minio.io/docs/minio-client-quickstart-guide).
+### <a name="test-the-configuration"></a>3. Test the Configuration
+To test this configuration, access the Minio server via a browser or invoke it as described in the [Minio Client Quickstart Guide](https://docs.minio.io/docs/minio-client-quickstart-guide).
```
Comment (eco-minio) [inline docs/distributed/README.md]: Do we need to mention how to check the browser?  Since we don't give an example of server startup output new users might not know how

**Review by eco-minio — CHANGES_REQUESTED** [review]

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :x: [more...](#fe21cf72-3c0e-43e9-b578-79b42f249875) |
| mint-xl.sh | :x: [more...](#6043f5d6-cf75-46bd-a682-945422de1101) |
| mint-large-bucket.sh | :x: [more...](#c1ea881a-3edc-4ea4-b9ed-fbd81261ba14) |
| mint-dist-xl.sh | :x: [more...](#f87c67da-8ea6-413a-b5cc-71049dd9751a) |
<a name="f87c67da-8ea6-413a-b5cc-71049dd9751a"></a>
### 6842-b8c84ff/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      minikube:30479
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 5f94d43a5812:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 35 seconds
{
  "name": "awscli",
  "duration": 17573,
  "function": "aws --endpoint-url http://minikube:30479 s3api get-object --bucket awscli-mint-test-bucket-3672 --key awscli-mint-test-bucket-3672-object /tmp/datafile-0-b\n",
  "status": "FAIL",
  "error": "An error occurred (InternalError) when calling the GetObject operation (reached max retries: 4): We encountered an internal error, please try again."
}

Executed 0 out of 13 tests successfully.
```
<a name="c1ea881a-3edc-4ea4-b9ed-fbd81261ba14"></a>
### 6842-b8c84ff/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      minikube:30614
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 4db25c8c409b:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 36 seconds
{
  "name": "awscli",
  "duration": 15588,
  "function": "aws --endpoint-url http://minikube:30614 s3api get-object --bucket awscli-mint-test-bucket-19773 --key awscli-mint-test-bucket-19773-object /tmp/datafile-0-b\n",
  "status": "FAIL",
  "error": "An error occurred (InternalError) when calling the GetObject operation (reached max retries: 4): We encountered an internal error, please try again."
}

Executed 0 out of 13 tests successfully.
```
<a name="6043f5d6-cf75-46bd-a682-945422de1101"></a>
### 6842-b8c84ff/mint-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      minikube:32418
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp ae3130b6641e:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 1 minutes and 19 seconds
{
  "name": "awscli",
  "duration": 28497,
  "function": "aws --endpoint-url http://minikube:32418 s3api get-object --bucket awscli-mint-test-bucket-2594 --key awscli-mint-test-bucket-2594-object /tmp/datafile-0-b\n",
  "status": "FAIL",
  "error": "An error occurred (InternalError) when calling the GetObject operation (reached max retries: 4): We encountered an internal error, please try again."
}

Executed 0 out of 13 tests successfully.
```
<a name="fe21cf72-3c0e-43e9-b578-79b42f249875"></a>
### 6842-b8c84ff/mint-compression-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      minikube:30435
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp fb6601434cad:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 1 minutes and 20 seconds
{
  "name": "awscli",
  "duration": 29799,
  "function": "aws --endpoint-url http://minikube:30435 s3api get-object --bucket awscli-mint-test-bucket-30469 --key awscli-mint-test-bucket-30469-object /tmp/datafile-0-b\n",
  "status": "FAIL",
  "error": "An error occurred (InternalError) when calling the GetObject operation (reached max retries: 4): We encountered an internal error, please try again."
}

Executed 0 out of 13 tests successfully.
```

File: docs/distributed/README.md [hunk 8]
```diff
@@ -1,74 +1,86 @@
...
 
-A stand-alone Minio server would go down if the server hosting the disks goes offline. In contrast, a distributed Minio setup with _n_ disks will have your data safe as long as _n/2_ or more disks are online. You'll need a minimum of _(n/2 + 1)_ [Quorum](https://github.com/minio/dsync#lock-process) disks to create new objects though.
+### High Availability
 
-For example, an 8-node distributed Minio setup with 1 disk per node would continue serving files, even if up to 4 disks are offline. But, you'll need at least 5 disks online to create new objects.
+A standalone server running Minio Server will go down if the hard drives go offline. In contrast, a distributed Minio configuration with N disks will keep data safe as long as N/2 or more hard drives are online. This requires a minimum of (N/2 + 1) [Quorum](https://github.com/minio/dsync#lock-process) hard drives to create new objects.
```
Comment (nitisht) [inline docs/distributed/README.md]: > N disks will keep data safe as long as N/2 or more hard drives are online

We can finalise on one of `disk` or `drive` and use that uniformly. I think `drive` is better as it refers the whole spectrum from HDDs to NVMes

File: docs/distributed/README.md [hunk 9]
```diff
@@ -1,74 +1,86 @@
...
 
-Minio in distributed mode lets you pool multiple drives (even on different machines) into a single object storage server. As drives are distributed across several nodes, distributed Minio can withstand multiple node failures and yet ensure full data protection.
+When running Minio in distributed mode (_distributed Minio_), multiple drives from different machines can be pooled into a single namespace. As drives are distributed across several nodes, distributed Minio ensures full data protection when multiple node failures occur.
 
-## Why distributed Minio?
+## Why Distributed Minio?
 
-Minio in distributed mode can help you setup a highly-available storage system with a single object storage deployment. With distributed Minio, you can optimally use storage devices, irrespective of their location in a network.
+Running Minio in distributed mode allows for the deployment of a high-availability, object storage system, enabling optimal use of storage devices regardless of their location on a network.
```
Comment (nitisht) [inline docs/distributed/README.md]: > deployment of a high-availability

IMO we can call it as `deployment of a highly-available`

File: docs/distributed/README.md [hunk 10]
```diff
@@ -1,74 +1,86 @@
...
 
-### Data protection
+**Note:** Each cluster must reside in the same network, but multiple clusters from multiple networks can be federated.
```
Comment (nitisht) [inline docs/distributed/README.md]: We can probably link to the federation guide here https://docs.minio.io/docs/minio-federation-quickstart-guide.html

File: docs/distributed/README.md [hunk 11]
```diff
@@ -1,74 +1,86 @@
...
 
-A stand-alone Minio server would go down if the server hosting the disks goes offline. In contrast, a distributed Minio setup with _n_ disks will have your data safe as long as _n/2_ or more disks are online. You'll need a minimum of _(n/2 + 1)_ [Quorum](https://github.com/minio/dsync#lock-process) disks to create new objects though.
+### High Availability
 
-For example, an 8-node distributed Minio setup with 1 disk per node would continue serving files, even if up to 4 disks are offline. But, you'll need at least 5 disks online to create new objects.
+A standalone server running Minio Server will go down if the hard drives go offline. In contrast, a distributed Minio configuration with N disks will keep data safe as long as N/2 or more hard drives are online. This requires a minimum of (N/2 + 1) [Quorum](https://github.com/minio/dsync#lock-process) hard drives to create new objects.
+
+For example, an 8-node distributed Minio configuration with one hard drive per node will continue serving files even if as many as 4 hard drives are offline. This configuration requires at least 5 hard drives online to create new objects.
```
Comment (nitisht) [inline docs/distributed/README.md]: > with one hard drive

Should be `with one drive` . As we recommend fast NVMe / flash / SSDs, using the term hard drive is not a good idea in this document

File: docs/distributed/README.md [hunk 12]
```diff
@@ -1,74 +1,86 @@
...
 
 ### Limits
 
-As with Minio in stand-alone mode, distributed Minio has a per tenant limit of minimum 2 and maximum 32 servers. There are no limits on number of disks shared across these servers. If you need a multiple tenant setup, you can easily spin up multiple Minio instances managed by orchestration tools like Kubernetes.
+Distributed Minio supports 2-32 servers per tenant. There are no limits on the number of hard drives shared across these servers and nor does Minio impose a limit to the number of hard drives used. For a multi-tenant configuration, orchestrate multiple Minio instances using a tool like Kubernetes.
```
Comment (nitisht) [inline docs/distributed/README.md]: Suggest rewording to something like this

`Distributed Minio requires at least 2 nodes and supports upto 32 nodes. There are no limits on the number of drives on each server, but total number of drives in the deployment should be even. For a multi-tenant configuration, orchestrate multiple Minio instances using a tool like Kubernetes.`

File: docs/distributed/README.md [hunk 13]
```diff
@@ -1,74 +1,86 @@
...
 
-Note that with distributed Minio you can play around with the number of nodes and drives as long as the limits are adhered to. For example, you can have 2 nodes with 4 drives each, 4 nodes with 4 drives each, 8 nodes with 2 drives each, 32 servers with 24 drives each and so on.
+**Note:** The of number of nodes in a Distributed Minio configuration must be within the limits. For example, a configuration can have 2 nodes with 4 hard drives each, 4 nodes with 4 hard drives each, 8 nodes with 2 hard drives each, 32 servers with 24 hard drives each, etc.
```
Comment (nitisht) [inline docs/distributed/README.md]: > The of number of nodes

Change to `The number of nodes`

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :x: [more...](#1d27f2b7-b2a9-4fd3-bce8-400f3b17357f) |
| mint-xl.sh | :x: [more...](#e86f6310-36f6-441d-9576-4b3da7243665) |
| mint-large-bucket.sh | :x: [more...](#9d6589e1-9a21-4f82-9b7a-6b994ae09693) |
| mint-dist-xl.sh | :x: [more...](#6e86a091-fcf4-464c-acbb-b6639781f3aa) |
<a name="1d27f2b7-b2a9-4fd3-bce8-400f3b17357f"></a>
### 6842-62d86f9/mint-compression-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      minikube:30848
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 9a8b14de717d:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 1 minutes and 20 seconds
{
  "name": "awscli",
  "duration": 28487,
  "function": "aws --endpoint-url http://minikube:30848 s3api get-object --bucket awscli-mint-test-bucket-26007 --key awscli-mint-test-bucket-26007-object /tmp/datafile-0-b\n",
  "status": "FAIL",
  "error": "An error occurred (InternalError) when calling the GetObject operation (reached max retries: 4): We encountered an internal error, please try again."
}

Executed 0 out of 13 tests successfully.
```
<a name="6e86a091-fcf4-464c-acbb-b6639781f3aa"></a>
### 6842-62d86f9/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      minikube:32165
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 785b7d819b8d:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 32 seconds
{
  "name": "awscli",
  "duration": 15412,
  "function": "aws --endpoint-url http://minikube:32165 s3api get-object --bucket awscli-mint-test-bucket-25257 --key awscli-mint-test-bucket-25257-object /tmp/datafile-0-b\n",
  "status": "FAIL",
  "error": "An error occurred (InternalError) when calling the GetObject operation (reached max retries: 4): We encountered an internal error, please try again."
}

Executed 0 out of 13 tests successfully.
```
<a name="9d6589e1-9a21-4f82-9b7a-6b994ae09693"></a>
### 6842-62d86f9/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      minikube:30122
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp e5a51b370805:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 38 seconds
{
  "name": "awscli",
  "duration": 16676,
  "function": "aws --endpoint-url http://minikube:30122 s3api get-object --bucket awscli-mint-test-bucket-2727 --key awscli-mint-test-bucket-2727-object /tmp/datafile-0-b\n",
  "status": "FAIL",
  "error": "An error occurred (InternalError) when calling the GetObject operation (reached max retries: 4): We encountered an internal error, please try again."
}

Executed 0 out of 13 tests successfully.
```
<a name="e86f6310-36f6-441d-9576-4b3da7243665"></a>
### 6842-62d86f9/mint-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      minikube:31851
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp bdf0f7c23e08:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 1 minutes and 16 seconds
{
  "name": "awscli",
  "duration": 25778,
  "function": "aws --endpoint-url http://minikube:31851 s3api get-object --bucket awscli-mint-test-bucket-30835 --key awscli-mint-test-bucket-30835-object /tmp/datafile-0-b\n",
  "status": "FAIL",
  "error": "An error occurred (InternalError) when calling the GetObject operation (reached max retries: 4): We encountered an internal error, please try again."
}

Executed 0 out of 13 tests successfully.
```

**Review by nitisht — APPROVED** [review]

File: docs/distributed/README.md (hunk 1 above)
Comment (eco-minio) [reply docs/distributed/README.md]: Per conversation with @deekoder this is fine

File: docs/distributed/README.md [hunk 14]
```diff
@@ -1,74 +1,86 @@
...
+#### Example
 
-Example 1: Start distributed Minio instance on 8 nodes with 1 disk each mounted at `/export1` (pictured below), by running this command on all the 8 nodes:
-![Distributed Minio, 8 nodes with 1 disk each](https://github.com/minio/minio/blob/master/docs/screenshots/Architecture-diagram_distributed_8.jpg?raw=true)
-#### GNU/Linux and macOS
+This example demonstrates how to start distributed Minio instances on eight nodes, with one drive each mounted at `/export1`: 
+
+![Distributed Minio, 8 nodes with 1 drive each](https://github.com/minio/minio/blob/master/docs/screenshots/Architecture-diagram_distributed_8.jpg?raw=true)
+
+Run the following command on all eight nodes:
```
Comment (eco-minio) [inline docs/distributed/README.md:68]: @belvid will change the graphic from 11 - 18 to 1 - 8
